React-Core.podspec

```diff
@@ -63,14 +63,14 @@ Pod::Spec.new do |s|
   end
 
   s.subspec "Hermes" do |ss|
-    ss.platforms = { :osx => "10.14" }
+    ss.platforms = { :osx => "10.14", :ios => "10.0" }
     ss.source_files = "ReactCommon/hermes/executor/*.{cpp,h}",
                       "ReactCommon/hermes/inspector/*.{cpp,h}",
                       "ReactCommon/hermes/inspector/chrome/*.{cpp,h}",
                       "ReactCommon/hermes/inspector/detail/*.{cpp,h}"
     ss.pod_target_xcconfig = { "GCC_PREPROCESSOR_DEFINITIONS" => "HERMES_ENABLE_DEBUGGER=1" }
     ss.dependency "RCT-Folly/Futures"
-    ss.dependency "hermes", "~> 0.6.0"
+    ss.dependency "hermes-engine"
```

Comment:
Do we need it here tho? I thought that this will resolve to whatever is already present. And the version is hardcoded in `scripts/react_native_pods.rb`, where e.g. Flipper version is set too.

Reply:
Gotcha. This is normally _conceptually_ the right place, but for now having it in `scripts/react_native_pods.rb` would suffice and ease work for maintainers, so I’m fine with that 👍